pre-docs/writing_plans.md

```diff
@@ -292,7 +292,7 @@ $r.each |$result| {
 
 ## Passing sensitive data to tasks
 
-Tasks can define parameters to be `Sensitive`, meaning the parameter values will not be logged by Bolt.
+Tasks can define parameters to be `sensitive`, meaning the parameter values will not be logged by Bolt.
```

Comment:
Should we note somewhere that you need to explicitly unwrap `Sensitive` types before passing to `run_task`?

Reply:
Added a sub-section below this.